README.md

```diff
@@ -0,0 +1,100 @@
+# Doka Labeler
+
+Automatically label new pull requests based on:
+* files being added, deleted, renamed or modified
+* assignees
+* front matter of your markdown files
+* your custom rules
+
+## Example
+
+Set "design review" label if a new HTML file is added to the _src_ folder:
+
+```yml
+"design review":
+  files:
+    added: src/**/*.html
+```
+
+## Getting Started
+
+Create `.github/labeler.yml` with a list of labels and conditions when they must be applied.
+
+The key is the name of the label in your repository (e.g. "refactor" or "design review needed"), the value is a set
+of conditions, described below. The label will be applied only when all conditions match (logical AND).
+
```

Comment:
```suggestion

For all labels there is an opportunity to setup the strategy of labelling. There are four options:

- `merge-with-others` — for merging the new list of labels after pushing and the old list before update (default - true);
- `completely-update` — for reset the list of labels after update;
-` add-if-not-exists` — add label if it doesn't exist before in the list of labels of repository;
- `remove-if-not-applicable` — remove the label after update if it is not applicable anymore.

Combine all strategies if it is applicable using the list of strategies. Put the list of these options as a key `strategy` within a label. If the value is not set in the list the value of key is false. Example:

```yaml
"feature":
   files:
     added: src/**/*.html
   strategy:
     - add-if-not-exists
     - completely-update
```

```